src/perform_step/kencarp_kvaerno_perform_step.jl

```diff
@@ -1718,3 +1722,375 @@ end
     @.. integrator.fsallast = z₈/dt
   end
 end
+
+
+@muladd function perform_step!(integrator, cache::KenCarp47ConstantCache, repeat_step=false)
+  @unpack t,dt,uprev,u,p = integrator
+  nlsolver = cache.nlsolver
+  @unpack γ,a31,a32,a41,a42,a43,a51,a52,a53,a54,a61,a62,a63,a64,a65,a73,a74,a75,a76,c3,c4,c5,c6 = cache.tab
+  @unpack α31,α32,α41,α42,α43,α51,α52,α61,α62,α63,α71,α72,α73,α74,α75,α76 = cache.tab
+  @unpack btilde3,btilde4,btilde5,btilde6,btilde7 = cache.tab
+  @unpack ea21,ea31,ea32,ea41,ea42,ea43,ea51,ea52,ea53,ea54,ea61,ea62,ea63,ea64,ea65,ea71,ea72,ea73,ea74,ea75,ea76 = cache.tab
+  @unpack eb3,eb4,eb5,eb6,eb7 = cache.tab
+  @unpack ebtilde3,ebtilde4,ebtilde5,ebtilde6,ebtilde7 = cache.tab
+  alg = unwrap_alg(integrator, true)
+
+  if typeof(integrator.f) <: SplitFunction
+    f = integrator.f.f1
+    f2 = integrator.f.f2
+  else
+    f = integrator.f
+  end
+
+  # precalculations
+  γdt = γ*dt
+
+  # calculate W
+  markfirststage!(nlsolver)
+
+  if typeof(integrator.f) <: SplitFunction
+    # Explicit tableau is not FSAL
+    # Make this not compute on repeat
+    z₁ = dt.*f(uprev, p, t)
+  else
+    # FSAL Step 1
+    z₁ = dt*integrator.fsalfirst
```

Comment:
I guess, we should use broadcasting in this branch as well?
```suggestion
    z₁ = dt .* integrator.fsalfirst
```